src/util/payee-name.js

```diff
@@ -0,0 +1,42 @@
+import { title } from './title/index.js';
+
+function formatPayeeIban(iban) {
+  return '(' + iban.slice(0, 4) + ' XXX ' + iban.slice(-4) + ')';
+}
+
+export const formatPayeeName = (trans) => {
+  const nameParts = [];
+
+  // get the correct name and account fields for the transaction amount
+  let name;
+  let account;
+  if (trans.amount > 0 || Object.is(Number(trans.amount), 0)) {
+    name = trans.debtorName;
+    account = trans.debtorAccount;
+  } else {
+    name = trans.creditorName;
+    account = trans.creditorAccount;
+  }
+
+  // use the correct fields for the amount if it was found
```

Comment:
The word `amount` in this comment seems to be wrong.  Maybe it should be `name`?